### PR TITLE
Add girderEnable method to factorize enabling/disabling inputs

### PR DESCRIPTION
**Pull request by vovythevov**
See Issue #1498

### CODE REVIEW

File: clients/web/src/utilities/jquery/girderEnable.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+ * Define jQuery plugins within this scope.
+ */
+(function ($) {
+    /**
+     * Helper selector to enable/disable inputs elements
+     *
+     * @param enable Whether to enable the element or not
+     */
+    $.fn.girderEnable = function (enable) {
+        if($(this).is(':input')){
```
Comment (msmolens) [inline clients/web/src/utilities/jquery/girderEnable.js]: Would it make sense to cache $(this) in a variable?

Comment (vovythevov) [inline clients/web/src/utilities/jquery/girderEnable.js]: Good catch. I'll change that.

**Comment by zachmullen** [comment]
@vovythevov could you re-open this PR against `2.0-integration`? Thanks!

**Comment by zachmullen** [comment]
Actually don't worry about this since you are on vacation; I'll cherry-pick it onto my current branch and close this out. Thanks!

**Pull request closed without merging**